app/dao/returned_letters_dao.py

```diff
@@ -50,6 +51,25 @@ def insert_or_update_returned_letters(references):
         db.session.connection().execute(stmt)
 
 
+def fetch_returned_letter_count(service_id):
```

Comment:
Optional: Was vaguely wondering about joining these two queries together so you only issue one query that returns both fields. Obviously this way is more general and could have more reuse but also YAGNI. I'm in two minds and don't feel strongly either way but just thought I'd raise it as something to consider if you hadn't already.